### ISSUE TITLE
Cannot add text or blocks to editor.

### ISSUE THREAD

**Issue by dwhagar** [WordPress/gutenberg#9147]
**Describe the bug**
Attempting to add anything other than a title fails every time.

**To Reproduce**
1. Go to 'Add Post'
2. Click on 'The + sign to add a block, or type to type in the existing block.'

**Expected behavior**
You should be able to start typing a new post.

**Error Text**
```
He@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/blocks/index.js?ver=1534522220:2:141858
value@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/editor/index.js?ver=1534522220:12:185745
Xf@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:91:274
lg@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:120:90
mg@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:120:388
gc@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:127:204
vb@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:232
ub@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:126:67
zd@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:124:451
ra@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:123:321
enqueueSetState@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react-dom.min.82e21c65.js:189:233
setState@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/vendor/react.min.ab6b06d4.js:18:456
https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:64702
forEach@[native code]
https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:60616
d@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:47107
https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:61726
w@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:38370
https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:38155
o@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:30435
c@https://home.blazingumbra.com/wordpress/wp-content/plugins/gutenberg/build/data/index.js?ver=1534522220:1:30634
promiseReactionJob@[native code]
```

**Screenshots**
![screen shot 2018-08-19 at 4 14 29 pm](https://user-images.githubusercontent.com/6186138/44312752-fb525b80-a3ca-11e8-98eb-3ef074ca0430.png)

**Desktop (please complete the following information):**
 - OS: MacOS X El Capitan (Cannot upgrade this further)
 - Browser Safari
 - Version 11.1.2

**Additional context**
- Gutenberg 3.6.2

**Comment by dwhagar** [comment]
I don't think I have anything similar to ACF installed.  I don't have a lot of plugins.

* Akismet Anti-Spam
* Invisible ReCaptcha
* Jetpack
* Notification
* Term Management Tools
* WP-Discourse

I clicked the "Copy Error" button to get the error, so that was all that was in the clipboard.

**Comment by designsimply** [comment]
Thanks for the extra information. I am unable to make the problem happen in testing and the plugins listed don't _seem_ like they would cause the error so we still need more information and I can make some extra troubleshooting recommendations to try to help.

Are you able to temporarily disable all other plugins then clear browser cache and try adding new blocks again? If it's still broken with Gutenberg as the only active plugin, then we can rule out a plugin conflict.

It's also possible for themes to present a conflict. Can you note your theme name and version for reference and also try testing after switching to a default theme such as Twenty Seventeen, clear browser cache, then check to see if the problem goes away?

**Comment by designsimply** [comment]
I installed and activated the following plugins:

1. Akismet Anti-Spam 4.0.8
1. Gutenberg 3.8.0
1. Invisible reCaptcha 1.2.1
1. Jetpack by WordPress.com 6.5
1. Notification 5.2.3
1. Term Management Tools 1.1.4
1. WP-Discourse 1.7.6

And as soon as I tried adding a block to a new post, I saw this error on screen:

> This block has encountered an error and cannot be previewed.

and these errors in the browser console:

> Block validation: Block validation failed because an error occurred while generating block content:
> TypeError: Object(...) is not a function
> index.js:2:143923

> TypeError: "Object(...) is not a function"
> react-dom.min.82e21c65.js:110:57

> TypeError: Object(...) is not a function
> index.js:12:100993

![screen shot 2018-09-17 at 4 41 04 pm](https://user-images.githubusercontent.com/1119271/45654406-bb999500-ba98-11e8-96f0-5f1eb83d22b1.png)
<sup>Seen at http://alittletestblog.com/wp-admin/post.php?post=15001&action=edit running WordPress 4.9.8 and Gutenberg 3.8.0 using Firefox 62.0 on macOS 10.13.6.</sup>

As soon as I uninstall Notification 5.2.3 then Gutenberg starts to work.

I reported the issue at https://wordpress.org/support/topic/cannot-add-blocks-to-posts-when-both-notifications-and-gutenberg-are-installed/
